package-lock.json

```diff
@@ -11,6 +11,9 @@
         "packages/*",
         "packages/components/*"
       ],
+      "dependencies": {
+        "@contentful/f36-density-container": "^4.0.0-alpha.3"
+      },
```

Comment:
Is this intentional?

Reply:
Ah good shout, I'll need to regenerate the lock file. Will do :)

Reply:
updated, refefences to `"@contentful/f36-density-container"` which were added in previous PR, are now removed, since we are using relative import now